int64 front
int64 frontRight
int64 right
int64 backRight
int64 back
int64 backLeft
int64 left
int64 frontLeft